lms/validation/__init__.py

```diff
@@ -64,6 +69,9 @@ def foo_view(request):
     "APIRecordSpeedgraderSchema",
     "APIReadResultSchema",
     "APIRecordResultSchema",
+    "PlainSchema",
+    "PyramidRequestSchema",
+    "RequestsResponseSchema",
```

Comment:
`lms/validation/__init__.py`'s `__all__` explicitly lists the names that make up `lms.validation`'s public interface. So here, I'm explicitly adding the base schemas to the public interface.